tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -95,51 +135,100 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
+        LOGGER.error("ORCA_DEFAULT_BUCKET environment variable is not set.")
         raise
 
     multipart_chunksize_mb_str = config.get(CONFIG_MULTIPART_CHUNKSIZE_MB_KEY, None)
     if multipart_chunksize_mb_str is None:
-        # TODO: Change this to a logging statement
-        multipart_chunksize_mb = int(os.environ['DEFAULT_MULTIPART_CHUNKSIZE_MB'])
-        print(f'{CONFIG_MULTIPART_CHUNKSIZE_MB_KEY} is not set for config. '
-              f'Using default value of {multipart_chunksize_mb}.')
+        multipart_chunksize_mb = int(os.environ["DEFAULT_MULTIPART_CHUNKSIZE_MB"])
+        LOGGER.debug(
+            "{CONFIG_MULTIPART_CHUNKSIZE_MB_KEY} is not set for config. Using default value of {multipart_chunksize_mb}.",
+            CONFIG_MULTIPART_CHUNKSIZE_MB_KEY=CONFIG_MULTIPART_CHUNKSIZE_MB_KEY,
+            multipart_chunksize_mb=multipart_chunksize_mb,
+        )
     else:
         multipart_chunksize_mb = int(multipart_chunksize_mb_str)
 
+    try:
+        metadata_queue_url = os.environ.get("METADATA_DB_QUEUE_URL")
+        if metadata_queue_url is None or len(metadata_queue_url) == 0:
+            raise KeyError("METADATA_DB_QUEUE_URL environment variable is not set.")
+    except KeyError:
+        LOGGER.error("METADATA_DB_QUEUE_URL environment variable is not set.")
+        raise
+
     granule_data = {}
     copied_file_urls = []
 
+    # initiate empty SQS body dict
+    sqs_body = {"provider": {}, "collection": {}, "granule": {}}
+    # 'providerName' set to None because Cumulus only returns providerId for now.
+    # In case it is available in the future, update orca_copy_to_glacier_workflow.asl.json and config.json as needed
+    sqs_body["provider"]["name"] = None
+    sqs_body["provider"]["providerId"] = config["providerId"]
+    sqs_body["collection"]["shortname"] = config["collectionShortname"]
+    sqs_body["collection"]["version"] = config["collectionVersion"]
+    # Cumulus currently creates collectionId by concating shortname + ___ + version 
+    # See https://github.com/nasa/cumulus-dashboard/blob/18a278ee5a1ac5181ec035b3df0665ef5acadcb0/app/src/js/utils/format.js#L342
+    sqs_body["collection"]["collectionId"] = (
+        config["collectionShortname"] + "___" + config["collectionVersion"]
+    )
     # Iterate through the input granules (>= 0 granules expected)
     for granule in granules_list:
         # noinspection PyPep8Naming
-        granuleId = granule['granuleId']
+        granuleId = granule["granuleId"]
         if granuleId not in granule_data.keys():
-            granule_data[granuleId] = {'granuleId': granuleId, 'files': []}
+            granule_data[granuleId] = {"granuleId": granuleId, "files": []}
+        # populate the SQS body for granules
+        sqs_body["granule"]["cumulusGranuleId"] = granuleId
+        granule["createdAt"] = "2021-10-08T19:24:07.605323Z" # remove this line once this card is complete https://bugs.earthdata.nasa.gov/browse/CUMULUS-2718
```

Comment:
Good afternoon, as CUMULUS-2718 is close to being approved I noticed that the format of `granule.createdAt` is in string format. After some internal discussion with the IX team there seems to be a preference to providing `granule.createdAt` in numeric format (milliseconds since the epoch) to be consistent with the [Cumulus API Granule schema](https://github.com/nasa/cumulus/blob/master/packages/api/models/schemas.js#L357-L359). What impact would it have on your ticket to switch from string to numeric format for `granule.createdAt`?

Reply:
hi this would include updating 
copy_to_glacier input schema, unit tests and some docs which would be doable. 

thoughts on this @AndrewDorn  @bhazuka ?

Reply:
Initial discussions chose this format to match [reconciliation reports](https://nasa.github.io/cumulus-api/#retrieve-reconciliation-report), but this can certainly be changed. @jaortega527, just to confirm, you only want the input to copy_to_glacier changed, not the [output](https://github.com/nasa/cumulus-orca/blob/dc9a36e4d3e4fa59f3b57f2a70a1653f4235b839/tasks/orca_catalog_reporting/schemas/output.json#L49) from the reporting lambda?

Reply:
I would defer to @markdboyd and @Jkovarik on that question.

Reply:
Hey @AndrewDorn , I was able to get some input/feedback from the rest of the team. The consensus is that to maintain consistency with the Cumulus API schema, we are requesting that all uses of `granule.createdAt` be in numeric format including output from the reporting lambda.